stable/artifactory-ha/README.md

```diff
@@ -710,6 +710,9 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.image.pullPolicy`       | Container pull policy                | `IfNotPresent`                             |
 | `artifactory.image.repository`       | Container image                      | `docker.bintray.io/jfrog/artifactory-pro`  |
 | `artifactory.image.version`          | Container image tag                  | `.Chart.AppVersion`                        |
+| `artifactory.priorityClass.create`   | Create a PriorityClass object        | `false`                                    |
+| `artifactory.priorityClass.value`    | Priority Class value                 | `1000000000`                               |
+| `artifactory.priorityClass.name`     | Priority Class name                  | `artifactory-priority-class`               |
```

Comment:
Shouldn’t this be based on the release name?

Reply:
Yes. Good point. I will update.